pages/about.html

```diff
@@ -0,0 +1,201 @@
+                            </div>
+                            <div class="close">
+                                <img src="assets/icons/close-icon.png"
+                                    class="mobile"
+                                    id="menu-close" />
+                            </div>
+                        </div>
+
+                    </div>
+
+                </nav>
+            </header>
+            <section class="hero">
+                <div class="headline">
+                    <p>Your help goes a long way!</p>
+                    <h1>Amanna Trust </h1>
+                </div>
+
+                <div class="card">
+                    <p>The Amanna Trust (AT) is a Christian charitable
+                        organization that was founded in 2015 (Reg. No. 2094 -
+                        PIN. P051557264R) on the premise that every child
+                        deserves access to post-primary education. Since the
+                        inception of Free Primary Education in 2003, the dreams
+                        of many children have been snuffed out because of their
+                        parents/guardians’ inability to afford secondary
+                        education. </p>
+                </div>
+
+                <p>The greatness of humanity is not in being human, but in being
+                    humane. – Mahatma Gandhi
+                </p>
+
+            </section>
+
+        </div>
+        <section class="objectives">
+            <h3>Our Objectives</h3>
+            <hr>
+            <div class="objectives-container">
+                <div class="bg">
+                    <ul class="objective">
+                        <li>
+                            <p>To promote the best interest of children
+                                including health, education and general well
+                                being.</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="bg">
+                    <ul class="objective" id="values">
+                        <li>
+                            <p>To support children to adulthood by facilitating
+                                their education and instilling in them life
+                                skills.
+                            </p>
+                        </li>
+
+                    </ul>
+
+                </div>
+
+                <div class="bg">
+                    <ul class="objective">
+                        <li>
+                            <p>
+                                To influence children positively and
+                                facilitate them to nature values that will
+                                influence their communities and society
+                                positively.
+                            </p>
+                        </li>
+                    </ul>
+
+                </div>
```

Comment:
Kindly stick to the given design only the content can change the idea is to replicate the design for a different topic the content of this section is about the logo refer to the images 

**your design**
![Capture d’écran du 2023-08-02 00-42-48](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/e840cd1b-1756-48fb-8446-cc0d57170bfc)

**template**
![Capture d’écran du 2023-08-02 00-42-10](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/a66c5d9a-f900-4b38-9ceb-e95fc1d2b271)